foodsaving/stores/models.py

```diff
@@ -30,6 +30,18 @@ def __str__(self):
         return '{} ({})'.format(self.name, self.group)
 
 
+class Feedback(BaseModel):
+    class Meta:
+        unique_together = ('weight', 'comment')
```

Comment:
You can delete `unique_together` (and the whole `class Meta`) as it would have unwanted effects. Remember our Slack chat about it?